airbyte-integrations/connectors/source-facebook-pages/source_facebook_pages/spec.json

```diff
@@ -0,0 +1,21 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/facebook-pages",
```

Comment:
By the way, this documentation is missing.

Reply:
I added a placeholder doc in https://github.com/airbytehq/airbyte/pull/5847. Please populate the content later on.